### PR TITLE
Update company believe

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
## Description

Per an internal discussion, the update company believe.

## Related Issue(s)

Supercedes: #2111

## Checklist

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

File: src/handbook/company/strategy.md [hunk 1]
```diff
@@ -5,30 +5,33 @@ meta:
...
-Our current (prospective) customers have already adopted Node-RED and need to
-professionalize their data streams and workflows. FlowFuse is the only company
-that can help them.
+FlowFuse will empower all engineers to innovate faster, by bringing the joy of software development to the world.
 
-### Our mission
+## Achieving our Mission
 
-FlowFuse exists to empower bottom-up innovation. We allow companies to professionalize their workflows to business critical applications.
+At the core of the product is Node-RED, the Low-Code platform that allows many
```
Comment (Hasmin-AC) [inline src/handbook/company/strategy.md:18]:
```suggestion
At the core of FlowFuse is Node-RED, the Low-Code platform that allows many
```

File: src/handbook/company/strategy.md [hunk 2]
```diff
@@ -5,30 +5,33 @@ meta:
...
+to build software with little or no training. FlowFuse will only be successful if
```
Comment (Hasmin-AC) [inline src/handbook/company/strategy.md:19]:
```suggestion
to build software with little or no coding knowledge. FlowFuse will only be successful if
```

File: src/handbook/company/strategy.md [hunk 3]
```diff
@@ -5,30 +5,33 @@ meta:
...
+to be achieved. Two good examples are
+1. FlowFuse Dashboard - Allowing to build Dashboards and interactive applications
+1. Project-Link - Linking Edge devices to the Cloud, enabling broadcasts and point-to-point connections
 
-Please see our [company values](/handbook/company/values/)
+A key differenciator for FlowFuse is our approach to licensing of our software.
+The core is open, free as in beer and as in speech. Our product is open for scrutany,
+usage, improvements, for the world. While there's a subset of the product
+propietery licensed, the source is available to read. We believe that Open Source
+Software plays a key part in education, reducing vendor lock-in, and community building.
```
Comment (knolleary) [inline src/handbook/company/strategy.md:34]:
```suggestion
A key differentiator for FlowFuse is our approach to licensing of our software.
The core is open, free as in beer and as in speech. Our product is open for scrutany,
usage, improvements, for the world. While there's a subset of the product
proprietary licensed, the source is available to read. We believe that Open Source
Software plays a key part in education, reducing vendor lock-in, and community building.
```

**Review by knolleary — APPROVED** [review]
Looks good - couple of typos spotted

**Pull request merged**